.gitignore

```diff
@@ -32,3 +32,6 @@ index.md
 # Visual Studio temporary files
 .vs/
 *.user
+
+# Automatic dependencies
```

Comment:
These should be generated under `build/`, which is already ignored. Right?

Reply:
I added it because I (somehow) had a `sir.d` appear in my directory.  Could be operator error, but anyway, I just added it by reflex.  We can remove if you want.

Reply:
Let's remove it and if you ever see it again we can stick it back in.

Reply:
Pushed.